### PR TITLE
Feature/match details page

### PR DESCRIPTION
**Pull request by sebastianwel**
Finished my 3rd [US](https://github.com/sebastianwel/MatchPal/issues/4). It contains a details page for matches using dynamic routing to go from the list of matches to a more detailed overview of the match and which bar shows this match. 

To be able to do this, I added two new mock-array, one for bars and their details and one for barsInMatches containing the barIds and which matches each bar shows. 
Created a /matches/[id] route to dynamically render the match-details. Added the required information as described in the US and rounded all up with a bit of styling. 

Please review and happy to receive your feedback 😄🤘

### CODE REVIEW

File: components/SelectedMatch/index.js [hunk 1]
```diff
@@ -0,0 +1,42 @@
+import styled from "styled-components"
+
+
+export default function SelectedMatch({date, time, homeTeam, homeTeamLogoColor, awayTeam, awayTeamLogoColor}){
+return(
+    <div>
+    <p style={{textAlign: "center"}}>{date}</p>
```
Comment (TmWgnr) [inline components/SelectedMatch/index.js:7]: not sure about inline styling.

Comment (sebastianwel) [reply components/SelectedMatch/index.js]: removed :D

Comment (TmWgnr) [reply components/SelectedMatch/index.js]: 👌🏼

File: pages/matches/[id].js [hunk 2]
```diff
@@ -25,21 +26,7 @@ console.log(currentMatch)
         <>
         <AppHeader/>
         <button style={{marginTop: 55}}>ᐸ</button>
```
Comment (TmWgnr) [inline pages/matches/[id].js]: what about this one? :D

**Review by TmWgnr — APPROVED** [review]
LGTM. 
good decision to switch inline styling to styled components. 
good luck for your next US :)

**Pull request merged**